### PR TITLE
[Sprite Lab] wall sprites don't push each other

### PR DESCRIPTION
**Pull request by mikeharv**
Sprites in the new Game Design lessons have a explicitly set `group` property in order simplify handling of sprite collisions, gravity, and jumping. It is desired that sprites in the `walls` group block all other sprites from moving and prevent them from overlapping. It's important that wall sprites not block other wall sprites or they will be pushed away. This adds the ability to get the list of sprites with no group so that the helper library can automatically have wall sprites block them. 

Before: 

After:

As the levels have been iterated upon, a couple other groups ("enemies", "objects") have been removed so I've cleaned up references to them in the helper library.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/config/libraries/zGameDev.interpreted.js [hunk 1]
```diff
@@ -1,32 +1,16 @@
...
 setDefaultSpriteSize(CELL_SIZE);
 
 // Handles sprite interactions every frame.
 function gameDevLoop() {
   collide('collide', {group: 'players'}, {group: 'walls'});
-  collide('collide', {group: 'objects'}, {group: 'walls'});
-  collide('collide', {group: 'enemies'}, {group: 'walls'});
-  collide("collide", ({costume: "all"}), ({group: "walls"}));
- 
+  collide('collide', {group: ''}, {group: 'walls'});
```
Comment (molly-moen) [inline dashboard/config/libraries/zGameDev.interpreted.js:10]: for my understanding: a group of `''` means the sprite has no group? How does making this change mean wall sprites don't collide with wall sprites (what was making that happen before)?

Comment (mikeharv) [reply dashboard/config/libraries/zGameDev.interpreted.js:10]: The `group` property is undefined for sprites unless explicitly set. We're using `''` as a way to signal that we want to return the ones without a group. What was doing it before was this removed line:

`collide("collide", {costume: "all"}, {group: "walls"});`

Which can be read "all sprites collide with walls". Now, it's just "all sprites without a group collide with walls".

Comment (mikeharv) [reply dashboard/config/libraries/zGameDev.interpreted.js:10]: It's currently only possible to set a value to `sprite.group` using a dropdown, and none of the options use an empty string.

Comment (molly-moen) [reply dashboard/config/libraries/zGameDev.interpreted.js:10]: thanks! It might be worth adding a comment explaining this.

**Review by molly-moen — APPROVED** [review]

File: dashboard/config/libraries/zGameDev.interpreted.js (hunk 1 above)
Comment (mikeharv) [reply dashboard/config/libraries/zGameDev.interpreted.js:10]: Will add!

**Pull request merged**